src/platform/plugins/shared/data/common/search/expressions/esql.ts

```diff
@@ -328,6 +329,13 @@ export const getEsqlFn = ({ getStartDependencies }: EsqlFnArguments) => {
           );
           const indexPattern = getIndexPatternFromESQLQuery(query);
 
+          const appliedTimeRange = input?.timeRange
+            ? {
+                from: DateMath.parse(input.timeRange.from),
+                to: DateMath.parse(input.timeRange.to),
```

Comment:
I think we need to round up here . . . right now the quick range of "Today" ends up being the same exact `from`/`to`.

```suggestion
                to: DateMath.parse(input.timeRange.to, { roundUp: true }),
```

https://github.com/user-attachments/assets/a409f3fb-20ad-4d36-ae72-19e8ba8beeb6

Reply:
resolved in https://github.com/elastic/kibana/pull/196049/commits/ec2506e006fad0108acde0460d3445589da483b7